internal/rbd/mirror.go

```diff
@@ -118,27 +118,33 @@ type imageMirrorStatus struct {
 	State       string `json:"state"` // rbd image state
 	Description string `json:"description"`
 	LastUpdate  string `json:"last_update"`
+	PeerSites   []struct {
```

Comment:
Can any of the field can be empty?

Reply:
AFAIK, No this cannot be empty as at least one peer site will be present in case of mirroring.